ha/util/conf_store.py

```diff
@@ -110,3 +121,93 @@ def set_cluster_cardinality(self, index):
         cluster_cardinality_key = CLUSTER_CARDINALITY_KEY
         cluster_cardinality_value = {CLUSTER_CARDINALITY_NUM_NODES: num_pods, CLUSTER_CARDINALITY_LIST_NODES : watch_pods }
         self._confstore.update(cluster_cardinality_key, json.dumps(cluster_cardinality_value))
+
+    def get_cvg_list(self, index, node_id):
```

Comment:
Method can be static_method. It would be appropriate as it is utils.

Reply:
Done.